test/fixtures.js

```diff
@@ -25,4 +25,22 @@ async function getSignersAndDeployContracts() {
 	return { admin, payer, payee1, payee2, streamManager, mockUSDT, maliciousToken };
 }
 
-module.exports = { getSignersAndDeployContracts }
\ No newline at end of file
+// Creating the stream
+async function createdOpenStream(payee) {
+  const { payer, streamManager, mockUSDT } = await loadFixture(getSignersAndDeployContracts)
+
+  const rate = 1500
+  const terminationPeriod = 18 * 24 * 3600; // 18 days
+  const cliffPeriod = 24 * 3600; // 24 hrs
+
+  // Creating stream
+  await streamManager.connect(payer).createOpenStream(
+    payee,
+    mockUSDT.address,
+    rate,
+    terminationPeriod,
+    cliffPeriod
+  );
```

Comment:
@CTAPCKPIM You can return values from the previous fixture and use this one.

e.g.

const { payer, streamManager, mockUSDT, ...rest} = await loadFixture(get..)

// createOpenStream
const streamId = // from events
return { streamId, payer, streamManager, mockUSDT, ..rest}

Reply:
@georgeciubotaru 
Do not quite understand

Reply:
`createdOpenStream(payee)` - should also return addresses and contract instances, so that I would write in the test itself:

```JavaScript
const { payer, payee1, streamManager, mockUSDT } = await loadFixture(createdOpenStream(payee.address))
```
But not:
```JavaScript
const { payer, payee1, streamManager, mockUSDT } = await loadFixture(getSignersAndDeployContracts)
// Creating stream
await createdOpenStream(payee1.address)
```
When I make two fixture calls